packages/expo-web-browser/android/src/main/java/expo/modules/webbrowser/WebBrowserModule.kt

```diff
@@ -171,42 +122,49 @@ class WebBrowserModule(context: Context?) : ExportedModule(context) {
       }
     } catch (ignored: IllegalArgumentException) {
     }
-    builder.setShowTitle(showTitle)
-    if (addDefaultShareMenuItem) {
+
+    builder.setShowTitle(options.showTitle)
+
+    if (options.enableDefaultShareMenuItem) {
       builder.addDefaultShareMenuItem()
     }
 
-    val intent = builder.build().intent
+    return builder.build().intent.apply {
+      // We cannot use builder's method enableUrlBarHiding, because there is
+      // no corresponding disable method and some browsers enables it by default.
```

Comment:
```suggestion
      // We cannot use the builder's method enableUrlBarHiding, because there is
      // no corresponding disable method and some browsers enable it by default.
```